packages/react/src/test/echoSchema.ts

```diff
@@ -0,0 +1,28 @@
+export const schema = {
```

Comment:
Don't put this in another file. This file is _really_ coupled to the echoDb.ts file, hardly other file will ever use it, so better to keep it private there.